tests/test_matrix_utils_auth.py

```
@@ -1028,15 +1042,15 @@ def test_get_e2ee_store_dir(mock_makedirs):
     mock_makedirs.assert_called_once()
 
 
-@patch("mmrelay.config.get_base_dir")
+@patch("mmrelay.config.get_credentials_path")
 @patch("os.path.exists")
 @patch("builtins.open")
 @patch("mmrelay.config.json.load")
 def test_load_credentials_success(
-    mock_json_load, mock_open, mock_exists, mock_get_base_dir
+    mock_json_load, mock_open, mock_exists, mock_get_credentials_path
```

Ruff (0.14.14)

[warning] Unused function argument: `mock_open`

(ARG001)